packages/core/helper-plugin/lib/src/components/RelationInput/RelationInput.js

```diff
@@ -49,10 +60,18 @@ export const RelationInput = ({
               components={{ Option }}
               options={searchResults.data.pages.flat()}
               isDisabled={disabled}
+              isLoading={searchResults.isLoading}
               error={error}
               inputId={id}
               isSearchable
               isClear
+              loadingMessage={() =>
```

Comment:
I think I'd prefer to pass you that prop from the outside into `RelationInput` so that we are consistent with the other labels. You can choose the name of the prop :)

Reply:
works for me! mmh for the naming I guess it depends on what I said above about the difference between entries/relations, but maybe that's just me confused by this!
could be `loadingRelationsMessage` if we use the same for both or `loadingEntriesMessage` if we differentiate them, wdyt?